pandas/core/base.py

```diff
@@ -877,6 +877,10 @@ def _values(self):
         """ the internal implementation """
         return self.values
 
+    @property
+    def empty(self):
+        return not self.size
```

Comment:
actually this might just work for all NDFrame.

can you also move the doc-string to here.